mwui-token-farm/src/scripts/formats/utility-class.ts

```diff
@@ -0,0 +1,34 @@
+import { utilities } from "../../utils/utility-classes";
```

Comment:
@Fuasmattn i think this whole utility thing is redundant and can be removed - it generated an empty index.css file

Reply:
Yes I think this is outdated for some time as it doesn't fit the token structure anymore. Was just an experiment anyway